### DISCUSSION TITLE
Still maintained?

### DISCUSSION THREAD

**Discussion by s34m** [r-darwish/topgrade#1025] [Q&A]
Is this project still maintained, because the last commit was 3 months ago. I want to contribute but if there is no active maintenance i don't want to waste time.

**Comment by MCOfficer** [comment]
I for one haven't seen any activity from @r-darwish for 2 months. I'd guess IRL things, but regardless, it seems unmaintained for the time being.

**Comment by s34m** [comment]
I have forked the repository and will start including open pull requests. Also I'll start working on different issues for now. If you don't mind, please do every pull request one time for this repository and one time for the fork till I get an answer by @r-darwish .

Fork is available at: https://github.com/DottoDev/topgrade
